### PR TITLE
[FIX] inventory: fix typo

### PR DESCRIPTION
**Pull request by Whenrow**

### CODE REVIEW

**Review by jirocha — APPROVED** [review]

**Comment by jirocha** [comment]
Moved this one forward for the small edit.

File: content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst
Comment (jues-odoo) [inline content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:23]: Hi! Can we make a quick change to this sentence:

`Let’s suppose there is one warehouse location *WH/Stock* and two sub-locations *WH/Stock/Vegetables* and *WH/Stock/Fruits*.`

I changed wording that led to a grammatical error.

**Review by jues-odoo — CHANGES_REQUESTED** [review]
Hi! Just a quick change before we make the approval. Thanks :)

**Comment by jcs-odoo** [comment]
Hello @Whenrow and @jues-odoo  :)

I would suggest targeting 13.0 since the page is exactly the same.
If a change can be applied to multiple versions, we target the oldest one so that fw-bot can create the forward-port pull requests :) 

Here, I'm afraid you can't simply rebase and change the target on the PR --> all code-owners would be pinged. It's not a biggie, but the best way is to close this PR, cherry-pick your commit on a 13.0 branch and create a new PR targeting 13.0.

Also, speaking of code-owners, the appropriate reviewers are tagged automatically. No need to request a review from odoo/content-doc or AntoineVDV manually at the creation of the PR. HOWEVER, once the reviewer has approved the PR, then, the reviewer can request a review manually from the group **odoo/content-doc** to merge the PR :) 

I hope this helps.

Cheers!

File: content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst
Comment (jirocha) [reply content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:23]: Yes! I was only staring at the single highlighted changes I forgot about the sentence preceding it.

**Review by jirocha — CHANGES_REQUESTED** [review]

**Comment by mart-e** [comment]
> Here, I'm afraid you can't simply rebase and change the target on the PR --> all code-owners would be pinged.

I believe you can avoid this by putting the PR back to draft (top right) before retargeting.
Also, please don't forget to update the .pot in the `locale/` directory.

**Comment by jcs-odoo** [comment]
> > Here, I'm afraid you can't simply rebase and change the target on the PR --> all code-owners would be pinged.
> 
> I believe you can avoid this by putting the PR back to draft (top right) before retargeting. Also, please don't forget to update the .pot in the `locale/` directory.

Ooooh that's clever :) Thanks @mart-e 
cc @AntoineVDV

**Comment by AntoineVDV** [comment]
I believe that requesting a review from `@content-doc` which is the parent team of all content teams pings *everyone* in that team. Please don't do that. Instead, let `C3POdoo` assign the correct codeowner (content) team when the PR is created. Then, that team can request a review from `@doc-review` when they're satisfied with the content changes.

cc @StraubCreative

File: content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst
Comment (AntoineVDV) [reply content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst:23]: @jirocha Do you want to handle this change, or we merge as-is?

**Comment by jcs-odoo** [comment]
@robodoo r+

**Comment by jcs-odoo** [comment]
> > Here, I'm afraid you can't simply rebase and change the target on the PR --> all code-owners would be pinged.
> 
> I believe you can avoid this by putting the PR back to draft (top right) before retargeting. Also, please don't forget to update the .pot in the `locale/` directory.

Hi @mart-e !
I am not sure what you mean by updating the .pot files. I just had a look at it, but I don't understand what's its use nor what we should do with it. Do you mean all doc PR should normally edit these files?

**Comment by AntoineVDV** [comment]
@robodoo retry

**Pull request closed without merging**
